src/intl/resolver.js

```diff
@@ -257,6 +257,9 @@ function* toString(entity) {
 }
 
 export function format(ctx, args, entity) {
+  if (entity === undefined) {
+    return ['', []];
+  }
```

Comment:
I added the guard in https://github.com/l20n/l20n.js/commit/0cbc5694bb6654f7bc135bf590c19b44364e6f44